### PR TITLE
feat(price-feeder): minimum gas price

### PR DESCRIPTION
**Pull request by vuong177**
## Description

- Add minium gas-price config
- Calculate and set feeAmount when broadcastTx
---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by robert-zaremba — APPROVED** [review]
utAck

**Comment by toteki** [comment]
@adamewozniak This seems good to me, but can you give it a thumbs up since it's your domain?

**Comment by adamewozniak** [comment]
Actually, the gas price default should be 0 uumee - votes are free due to our antehandler

File: price-feeder/price-feeder.example.toml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 gas_adjustment = 1
+gas_price = "0.0001uume"
```
Comment (toteki) [inline price-feeder/price-feeder.example.toml:2]:
```suggestion
gas_price = "0uumee"
```
Setting default to zero (also `uume` -> `uumee`)

**Comment by robert-zaremba** [comment]
The ante handler adjusts the fees anyway.

**Comment by adamewozniak** [comment]
@vuong177 are you still able to finish this PR? ❤️

**Pull request closed without merging**